src/compas_timber/elements/plate.py

```diff
@@ -206,6 +230,20 @@ def add_feature(self, feature):
         """
         self._features.append(feature)
 
+    def shape(self):
+        # type: () -> compas.geometry.Brep
+        """The shape of the plate before other features area applied.
+
+        Returns
+        -------
+        :class:`~compas.geometry.Brep`
+            The shape of the element.
+
+        """
+        plate_geo = Brep.from_loft([NurbsCurve.from_points(pts, degree=1) for pts in [self.outline_a, self.outline_b]])
```

Comment:
opt for tuple where collection is anyways immutable
```suggestion
        plate_geo = Brep.from_loft([NurbsCurve.from_points(pts, degree=1) for pts in (self.outline_a, self.outline_b)])
```